Bio/_utils.py

```diff
@@ -73,15 +74,15 @@ def run_doctest(target_dir: Optional[str] = None, *args: Any, **kwargs: Any) ->
     import doctest
 
     # default doctest options
-    default_kwargs = {"optionflags": doctest.ELLIPSIS}
-    kwargs.update(default_kwargs)
+    doctest_attributes: Dict[str, Any] = {"optionflags": doctest.ELLIPSIS}
+    doctest_attributes.update(kwargs)
 
     cur_dir = os.path.abspath(os.curdir)
 
     print("Running doctests...")
     try:
         os.chdir(find_test_dir(target_dir))
-        doctest.testmod(*args, **kwargs)
+        doctest.testmod(*args, **doctest_attributes)
```

Comment:
So you're doing three things here:

* Type annotation, good
* Renaming a variable, because...
* Avoiding editing the passed in ``kwargs`` argument

Given they way we're using the ``run_tests`` function, I don't think we need to worry about side effects by editing the passed ``kwargs``, but this is cleaner.

Reply:
> * Type annotation, good

It is because mypy said that it can not call doctest.testmod with Dict[str, int] without type annotation, so I did not have a change :(

> * Renaming a variable, because...

I needed to update default_kwargs, but it becomes not default, but some others...

> * Avoiding editing the passed in kwargs argument

Yep, it should not be written in this way, since before patch, it always set `optionflags` to ELLIPSIS even if in kwargs there was another `optionflags`